dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -34,10 +34,10 @@
     %p Edit Markdown:
 
     %textarea#level_markdown_textarea
-    #markdown-preview
+    #level_markdown_textarea_preview
 
   .markdown.teacher
     %p Edit Teacher Only Markdown:
 
     %textarea#level_teacher_markdown_textarea
-    #teacher-markdown-preview
+    #level_teacher_markdown_textarea_preview
```

Comment:
There's also value in just being consistent about using one or the other (which we clearly arent).